{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,52 +1,67 @@
 [tool.poetry]
+authors = ["{{ cookiecutter.author_name }}"]
+description = "{{ cookiecutter.description }}"
 name = "{{ cookiecutter.repo_name }}"
 version = "{{ cookiecutter.version }}"
-description = "{{ cookiecutter.description }}"
-authors = ["{{ cookiecutter.author_name }}"]
 
 include = ["{{ cookiecutter.repo_name }}/py.typed"]
 
+[tool.poetry.scripts]
+cli = 'cli:cli'
```

Comment:
use `{{ cookiecutter.repo_name }} = "{{ cookiecutter.repo_name }}.__main__:cli"` 

1. left side should be unique per repo or if you'll get clashes for commands
2. I suggest using __main__ because of that being a python standard: https://docs.python.org/3/library/__main__.html#main-py-in-python-packages